yarn.lock

```diff
@@ -2381,6 +2381,15 @@
     "@sentry/utils" "7.46.0"
     tslib "^1.9.3"
 
+"@sentry/core@^7.44.0":
```

Comment:
funky duplicate

Reply:
I think it's related to it being a dep for profiling, I'll move these to peer deps in the future, but I want to move profiling to the js repo so we can detect breaking changes in those packages